source/api/extension/navplace/index.md

```diff
@@ -90,7 +90,7 @@ A Feature object represents a spatially bounded thing.  Every Feature object is
 
 #### 2.1.3 Feature Collection
 
-A FeatureCollection object has a member with the name `features`.  The value of `features` is a JSON array. Each element of the array is a Feature object as defined above.  It is possible for this array to be empty, but when used in the context of this extension _SHOULD NOT_ be empty.  It is also possible for this array to be extremely complex.  `navPlace` is intended to connect resources with geographic areas.  These areas should be bounded discrete regions of the map akin to extents.  navPlace is not meant for geographic layouts at the township, city, country or continental level.  Such descriptive datasets are meant for processing algorithms to determine complicated geospatial outcomes or overlays.  This goes beyond what navPlace and software consuming navPlace intend to support.  
+A FeatureCollection object has a member with the name `features`.  The value of `features` is a JSON array. Each element of the array is a Feature object as defined above.  It is possible for this array to be empty, but when used in the context of this extension _SHOULD NOT_ be empty.  It is also possible for this array to be extremely complex.  `navPlace` is intended to connect resources with geographic areas.  These areas should be bounded discrete regions of the map akin to extents.  navPlace is not meant for geographic layouts at the township, city, country or continental level.  Such descriptive datasets are meant for processing algorithms to determine complicated geospatial outcomes or overlays.  This goes beyond what `navPlace` and software consuming `navPlace` intend to support.  
```

Comment:
Some discrepancy between "`navPlace` is intended" and "navPlace is not meant".   I would not hesitate to be a bit more verbose and use an exactly parallel construction, e.g.,   "The `navPlace` property is [not] intended..."

Reply:
Re: "geospatial outcomes"  is this a typo?

Reply:
Geospatial outcomes is not a typo, but it probably also isn't the right phrase.  "are for processing algorithms to determine geospatial analytical results or widespread (overlays/layouts)" might be better.